plugins/services/src/js/reducers/serviceForm/JSONReducers/resourceLimits.ts

```diff
@@ -0,0 +1,46 @@
+function resourceLimitReducer(resourceField: string, parseFn = parseInt) {
+  return function(
```

Comment:
i don't understand what's happening in this function. can we rewrite it for more clarity?

Reply:
Would it help if there would be a comment to explain what the function is trying to achive?